drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -391,6 +391,7 @@ float AudioDriverPulseAudio::get_latency() {
 }
 
 void AudioDriverPulseAudio::thread_func(void *p_udata) {
+	Thread::set_name("Audio Pulse");
```

Comment:
The driver name is PulseAudio:

```suggestion
	Thread::set_name("Audio PulseAudio");
```